docs/deployment.md

```diff
@@ -4,18 +4,43 @@ short_title: Deployment
 description: Deploy your MyST site to static HTML or a MyST-aware server.
 ---
 
-The default themes for MyST sites are applications that render structured data _dynamically_, and are not static HTML sites. This choice allows the websites to include many [performance enhancements](./accessibility-and-performance.md) such as pre-fetching for instant page-transitions, loading indicators, and smaller network payloads. However, these advantages require that your website either (a) requires a web server or service that understands MyST sites; or (b) is changed to an HTML export that does not include these features but does allow you to host static files on services like GitHub pages.
+There are two different kinds of MyST website that can be deployed out of the box:
+- Statically generated sites, which can be served from static site hosts (like GitHub Pages).
+- Dynamically rendered sites, which use a MyST server.
+
+The high-level differences between these approaches are outlined in [](#deployment-comparison).
+:::{table} High-Level Comparison of MyST Site Deployment Types
+:label: deployment-comparison
+
+| Deployment Type | High Performance | Seamless Updates | Static Hosting |
+|:---------------:|:----------------:|:----------------:|:--------------:|
+|  Static website |         ❌        |         ❌        |        ✅       |
+|   MyST server   |         ✅        |         ✅        |        ❌       |
+:::
+
+The [default themes](website-templates.md#themes-bundled-with-myst) for MyST are designed to be MyST servers. Sites deployed in this manner benefit from [performance enhancements](./accessibility-and-performance.md) such as pre-fetching for instant page-transitions, loading indicators, and smaller network payloads. As the MyST themes receive new updates, it is easy to upgrade existing websites that have been deployed upon MyST servers, as the MyST site _content_ is only ever built once.
```

Comment:
```suggestion
:::{note} MyST was designed to be deployed as an application
Deploying MyST as an application has many benefits. For example, [performance enhancements](./accessibility-and-performance.md) (like pre-fetching for instant page-transitions, loading indicators, and smaller network payloads) and easier upgrades as new MyST versions are released.

The [default themes](website-templates.md#themes-bundled-with-myst) for MyST are designed to be MyST applications rather than static sites, but the core functionality is equally shared between the two options.
:::
```